.circleci/config.yml

```
@@ -240,24 +267,27 @@ jobs:
       - image: cimg/node:16.15
```

We can update this to be the same used in the solidity repo.
```suggestion
      - image: cimg/node:18.16
```

---

I merged #716 so versions are up to date now.